terraform/modules/puppet_ca/templates/puppet.tpl

```diff
@@ -31,7 +31,7 @@ runcmd:
   - r10k deploy environment -p --verbose -c /etc/puppetlabs/r10k.yaml
   - echo -e "nameserver 173.245.58.51\nnameserver 8.8.8.8" > /etc/resolv.conf
   - until /opt/puppetlabs/puppet/bin/puppet agent -t --environment=develop --server=puppetserver-0.${domain}; do echo "puppet failed, retry in 10 seconds"; sleep 10; done
-  - until grep "@8600" /etc/unbound/unbound.conf; do echo "Consul still not configured, retry in 10 seconds"; sleep 10; /opt/puppetlabs/puppet/bin/puppet agent -t --environment=develop --server=puppetserver-0.briggs.lan; done
+  - until grep "@8600" /etc/unbound/unbound.conf; do echo "Consul still not configured, retry in 10 seconds"; sleep 10; /opt/puppetlabs/puppet/bin/puppet agent -t --environment=develop --server=puppetserver-0.${domain}; done
```

Comment:
I'm not an expert on this stuff but isn't this still a terraform variable at this stage (see the previous line of code that uses ${domain}, is it not the same as that?)